Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -791,6 +793,7 @@ bool RenderLayerCompositor::updateCompositingLayers(CompositingUpdateType update
 #if !LOG_DISABLED || ENABLE(RESOURCE_USAGE)
     if (compositingLogEnabled() && isFullUpdate && (needHierarchyUpdate || needGeometryUpdate)) {
         MonotonicTime endTime = MonotonicTime::now();
+        UNUSED_PARAM(endTime);
```

Comment:
Same case as the previous frame and isMainFrame vars.

Reply:
done